### PR TITLE
[14.0] Add module purchase_order_receipt_status

### PR DESCRIPTION
**Pull request by grindtildeath**
This module adds a receipt status computed field on purchase order.

### CODE REVIEW

**Comment by yostashiro** [comment]
@grindtildeath I know the implementation is different but there is already this module https://github.com/OCA/purchase-workflow/tree/14.0/purchase_reception_status, just in case you were not aware.

File: purchase_order_receipt_status/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "version": "14.0.1.0.0",
+    "category": "Purchase",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "purchase_stock",
+    ],
+    "data": [
+        # "security/some_model_security.xml",
```
Comment (JuMiSanAr) [inline purchase_order_receipt_status/__manifest__.py:15]: Do we need this `data` entry?

**Review by JuMiSanAr — APPROVED** [review]

**Review by sebalix — COMMENTED** [review]
We should try to fix/complete the existing module if it doesn't fulfill our need indeed.

**Comment by grindtildeath** [comment]
@yostashiro Thank you. I don't know how I couldn't spot this module after searching extensively for something similar :confused: 

@sebalix Definitely agree, just not sure how to convince the customer yet :sweat_smile: 
(and we probably won't have the budget for it :disappointed: )

**Comment by grindtildeath** [comment]
Closing for now in favour of [purchase_reception_status](https://github.com/OCA/purchase-workflow/tree/14.0/purchase_reception_status)

**Pull request closed without merging**